ops/mainnet/prod/core/config.tf

```diff
@@ -426,6 +558,24 @@ locals {
       "1818848877" = {
         providers = ["https://linea-mainnet.infura.io/v3/${var.infura_key}", "https://rpc.linea.build"]
       }
+      "2053862243" = {
```

Comment:
We should use private RPC endpoints for all these chains.
And create tf variables for them

Reply:
gotcha!

Reply:
https://github.com/connext/monorepo/pull/5619/commits/a6e1743170637c9e7604354f0186f74e57fc476c